web/app/js/components/Namespace.jsx

```diff
@@ -117,8 +117,8 @@ class Namespaces extends React.Component {
     }
     return (
       <div className="page-section">
-        <Typography variant="h5">{friendlyTitle(resource).plural}</Typography>
```

Comment:
Minor nit, you no longer need the `Typography` library so you don't need to define it on line 9.